TACC/tup-ui
# test(tup-cms): build ui demo if core-cms will not 🧪

## Overview

Build the UI demo, so:
- Core-CMS need not build it for sites that don't use it.
- TUP CMS can apply its custom styles.

<details><summary>Why?</summary>

Core-CMS images take very long to build.

One of the largest layers is building the demo.

The demo is only used on TUP CMS and CEP.

I propose only building it for sites that load it.

</details>

> **Important**
> **Closed**, because it adds too much time to the TUP CMS build.

## Related

- requires https://github.com/TACC/Core-CMS/pull/705

## Changes

- **changed** core-cms image tag
- **added** ui demo build step to dockerfile

## Testing

1. Verify build succeeds.
2. Verify demo is accessible e.g. https://dev.tup.tacc.utexas.edu/static/ui/.

<details>

- build: https://jenkins01.tacc.utexas.edu/job/tup-cms-deploy/107
- changes: https://github.com/TACC/tup-ui/pull/307

</details>

## UI

| Dev |
| - |
| ![Screenshot 2023-08-28 at 2 59 43 PM](https://github.com/TACC/tup-ui/assets/62723358/318a2548-58fb-4f9f-9ccc-16ca42b7eff3) |
